test/functions_test.py

```diff
@@ -212,6 +217,34 @@
 15.9
 >>> params["digits"]
 1
+>>> params["cups"]
+1
+>>> args = parser.parse_args(["--method", 'steep-and-release', "--digits", '1', "--cups", '3'])
+>>> params = load_params(args)
+>>> params["coffee"] = calc_coffee(params)
+>>> params["water"]
+255
+>>> params["coffee"]
+47.8125
```

Comment:
Why `digits=1` while we have `47.8125`?

Reply:
It's not a bug. Rounding applies in the `filter_params` function, not in the `calc_coffee` function.